chat-server/src/integrations/openai.ts

```diff
@@ -0,0 +1,41 @@
+// TODO: Consider refactoring to use new Azure OpenAI client SDK - https://www.npmjs.com/package/@azure/openai
+import axios from "axios";
+import { CreateEmbeddingResponse } from "openai";
+interface CreateEmbeddingParams {
+  input: string;
+  user: string;
+}
+export class OpenAIClient {
```

Comment:
Minor, but, I think, as a rule, it's easier just to stick with CamelCase that treats acronyms and abbreviations as words...
```suggestion
export class OpenAiClient {
```

Reply:
will refactor

Reply:
updated all the `{O|o}pen{AI|ai}` -> `{O|o}penAi`

Reply:
did the class implementing interface. i'm partial to classes + constructors plus got them in some other places, so would like to keep (at least for now)